bootstrapvz/base/bootstrapinfo.py

```diff
@@ -131,6 +138,12 @@ def filter_state(state):
 		state['__class__'] = self.__module__ + '.' + self.__class__.__name__
 		return state
 
+	def __parse_image_name(self, manifest, manifest_vars):
+		image_name = manifest.image['name'].format(**manifest_vars)
+		# Ensure that we do not use disallowed characters in image name
+		image_name = image_name.lower().replace('.', '-').replace(' ', '-')
```

Comment:
I don't think this should happen here. The allowed character set is very provider specific - I suggest enforcing allowable characters through the manifest schema instead, this way you wont surprise users either when they are looking for the completed image.